pkg/services/system_intakes.go

```diff
@@ -86,6 +90,35 @@ func NewSaveSystemIntake(
 		if !ok {
 			return &apperrors.UnauthorizedError{Err: err}
 		}
+		updatedTime := clock.New().Now().UTC()
```

Comment:
`clock` needs to be passed a parameter to `NewSaveSystemIntake`. That's the only way to pass a mock clock with `clock.NewMock()` in tests.